src/main/flight/pid.h

```diff
@@ -171,6 +171,15 @@ typedef struct pidProfile_s {
     int8_t auto_profile_cell_count;         // Cell count for this profile to be used with if auto PID profile switching is used
     uint8_t transient_throttle_limit;       // Maximum DC component of throttle change to mix into throttle to prevent airmode mirroring noise
     char profileName[MAX_PROFILE_NAME_LENGTH + 1]; // Descriptive name for profile
+
+    uint8_t idle_hz;                        // minimum motor speed enforced by integrating p controller
```

Comment:
Since this is an array parameter group you will have to increment the version of it.

Reply:
done